php_fpm/README.md

```diff
@@ -1,57 +1,60 @@
-# Php_fpm Integration
+# PHP-FPM Check
 
-## Overview
+# Overview
 
-Enable the PHP-FPM check to monitor the state of your FPM pool and track requests performance.
+The PHP-FPM check monitors the state of your FPM pool and tracks request performance.
 
-## Installation
+# Installation
 
-Install the `dd-check-php_fpm` package manually or with your favorite configuration manager.
+The PHP-FPM check is packaged with the Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) on any servers that use PHP-FPM. If you need the newest version of the check, install the `dd-check-php-fpm` package.
 
-## Configuration
+# Configuration
+
+Create a file `php_fpm.yaml` in the Agent's `conf.d` directory:
 
-1. Configure the Agent to connect to your FPM status endpoint (look at your pools definition).
-Edit conf.yaml:
 ```
 init_config:
 
-instances:
-  - # Get metrics from your FPM pool with this URL
-    status_url: http://localhost/status
-    # Get a reliable service check of you FPM pool with that one
-    ping_url: http://localhost/ping
-    # These 2 URLs should follow the options from your FPM pool
-    # See http://php.net/manual/en/install.fpm.configuration.php
-    #   * pm.status_path
-    #   * ping.path
-    # You should configure your fastcgi passthru (nginx/apache) to
-    # catch these URLs and redirect them through the FPM pool target
-    # you want to monitor (FPM `listen` directive in the config, usually
-    # a UNIX socket or TCP socket.
-    #
-    # Use this if you have basic authentication on these pages
-    # user: bits
-    # password: D4T4D0G
-    #
-    # Array of custom tags
-    # By default metrics and service check will be tagged by pool and host
-    # tags:
-    #   - instance:foo
+instances: 
+  - status_url: http://localhost/status # or whatever pm.status_path is set to in your PHP INI
+    ping_url: http://localhost/ping     # or whatever ping.path is set to in your PHP INI
+    ping_reply: pong                    # the reply to expect from ping; default is 'pong'
+#   user: <YOUR_USERNAME>     # if the status and ping URLs require HTTP basic auth
+#   password: <YOUR_PASSWORD> # if the status and ping URLs require HTTP basic auth
+#   http_host: <HOST>         # if your FPM pool is only accessible via a specific HTTP 
```

Comment:
Should this be `if your FPM pool is only accessible via a specific HTTP`__` vHost`__ ?